### PR TITLE
Iss318

### PR DESCRIPTION
**Pull request by igorhdl**
added gpu_util and pci_caps unit tests

### CODE REVIEW

File: rvs/CMakeLists.txt [hunk 1]
```diff
@@ -84,6 +85,11 @@ set(SOURCES
   ../src/rvsliblogger.cpp ../src/rvslognodebase.cpp ../src/rvslognoderec.cpp ../src/rvslognode.cpp ../src/rvslognodestring.cpp ../src/rvslognodeint.cpp
 )
 
+## define test sources
```
Comment (mlucinhdl) [inline rvs/CMakeLists.txt]: Move in tests.cmake

File: rvs/CMakeLists.txt [hunk 2]
```diff
@@ -73,6 +73,7 @@ add_compile_options(-DBUILD_VERSION_STRING="${BUILD_VERSION_STRING}")
 
 ## additional libraries
 set ( PROJECT_LINK_LIBS libdl.so "${YAML_LIB_DIR}/libyaml-cpp.a" libpthread.so)
+set ( PROJECT_TEST_LINK_LIBS ${PROJECT_LINK_LIBS} libpci.so)
```
Comment (mlucinhdl) [inline rvs/CMakeLists.txt]: move in tests.cmake

File: rvs/CMakeLists.txt [hunk 3]
```diff
@@ -84,6 +85,11 @@ set(SOURCES
   ../src/rvsliblogger.cpp ../src/rvslognodebase.cpp ../src/rvslognoderec.cpp ../src/rvslognode.cpp ../src/rvslognodestring.cpp ../src/rvslognodeint.cpp
 )
 
+## define test sources
+set(TEST_SOURCES
```
Comment (mlucinhdl) [inline rvs/CMakeLists.txt]: move in tests.cmake

File: rvs/CMakeLists.txt [hunk 4]
```diff
@@ -98,7 +104,7 @@ file(GLOB TESTSOURCES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} test/test*.cpp )
 message ( "TESTSOURCES ${TESTSOURCES}" )
 
 LINK_DIRECTORIES(${UT_LIB})
-add_executable(rvstest ${TESTSOURCES} ${SOURCES})
+add_executable(rvstest ${SOURCES})
```
Comment (mlucinhdl) [inline rvs/CMakeLists.txt]: Move in ctests.cmake

**Comment by igorhdl** [comment]
gtest warnings fix

**Comment by igorhdl** [comment]
cpp lint fix

**Pull request merged**